modules/oe_theme_content_publication/src/Plugin/PageHeaderMetadata/PublicationContentType.php

```diff
@@ -54,6 +111,11 @@ public function getMetadata(): array {
       ],
     ];
 
+    $timestamp = $entity->get('oe_publication_date')->date->getTimestamp();
```

Comment:
If there is no node summary, above we return the metadata and we don't get a chance to set the publication date. Please see NewsContentType as an example of essentially exact same thing.